linked-events.swagger.yaml

```diff
@@ -723,6 +732,9 @@ definitions:
       publisher:
         type: string
         description: Organization responsible for this event record.
+      accessible:
```

Comment:
What is this?

Reply:
This is boolean value to tell if the event is easily accessible. 
Not necessarily unobstructed access, but easily accessed event.
At least Tavastia events uses this.

Reply:
Field is not supported on the upstream project: https://github.com/City-of-Helsinki/linkedevents. Could you contribute this feature to upstream before suggesting change to this swagger-file?

Unsupported features can break clients using other versions of API, so i cannot accept this change.

Reply:
Deleted 'accessible' field for now